src/main/scala/beam/agentsim/agents/household/HouseholdActor.scala

```diff
@@ -457,7 +457,7 @@ object HouseholdActor {
           veh.manager = Some(self)
           veh.spaceTime = SpaceTime(homeCoord.getX, homeCoord.getY, 0)
           for {
-            ParkingInquiryResponse(stall, _) <- parkingManager ? ParkingInquiry(homeCoord, "home", 0.0, None, 0)
+            ParkingInquiryResponse(stall, _) <- parkingManager ? ParkingInquiry.apply(homeCoord, "home")
```

Comment:
Any reason to explicitly call `apply`? `apply` is inserted when you call as though it is a constructor.

```suggestion
            ParkingInquiryResponse(stall, _) <- parkingManager ? ParkingInquiry(homeCoord, "home")
```

Reply:
not really -> removed